src/Training.AirBnb.Clone.Frontend/src/infrastructure/router/index.ts

```diff
@@ -22,6 +22,11 @@ const router = createRouter({
           path: '/host/homes',
           name: 'HostHomes',
           component: () => import('../../common/views/HostHomes.vue')
+      },
+      {
+          path:'/listings/:listingId',
```

Comment:
is it working? I mean there is double dot in front of listingId! Or is it must be like this?

Reply:
yes it works, there were two different methods. They are: double dot  and writing the full path

Reply:
Ok, all of them fine